### PR TITLE
🐛  Add wasm files as binary artifacts

### PR DESCRIPTION
**Pull request by gabibguti**
#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

- [X] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the current behavior?

WebAssembly binary files, identified by `.wasm` extension, are not considered as binaries in the Binary-Artifacts check.

#### What is the new behavior (if this is a feature change)?**

- [X] Tests for the changes have been added (for bug fixes/features)

#### Which issue(s) this PR fixes

No related issue.

#### Special notes for your reviewer

Retrieved `simple.wasm` test binary from https://raw.githubusercontent.com/mdn/webassembly-examples/master/js-api-examples/simple.wasm.

[Scorecard API](https://api.securityscorecards.dev/projects/github.com/nodejs/node) identifies `.o` and `.zip` as binaries but no `.wasm` binaries for nodejs/node.
```
{"name":"Binary-Artifacts","score":7,"reason":"binaries present in source code","details":["Warn: binary detected: deps/v8/third_party/ittapi/include/fortran/posix/x86/ittfortran.o:1","Warn: binary detected: deps/v8/third_party/ittapi/include/fortran/posix/x86_64/ittfortran.o:1","Warn: binary detected: deps/zlib/google/test/data/evil_via_invalid_utf8.zip:1"],"documentation":{"short":"Determines if the project has generated executable (binary) artifacts in the source repository.","url":"https://github.com/ossf/scorecard/blob/c61f6bc297ee71dc4b5f2511144d6c031b946089/docs/checks.md#binary-artifacts"}}
```
[While there are wasm binaries.](https://github.com/nodejs/node/search?q=extension%3Awasm)

#### Does this PR introduce a user-facing change?

> For user-facing changes, please add a concise, human-readable release note to
the `release-note`

> (In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note
Add wasm files as binary artifacts
```

The changes may impact the users score. The remediation steps can remain the same for Binary-Artifacts. Therefore, I understand no release note is needed.

### CODE REVIEW

File: cron/internal/data/projects.csv [hunk 1]
```diff
@@ -1243372,8 +1243372,8 @@ github.com/zowe/explorer-uss,num_dependents_deps.dev:0
 github.com/zowe/file-transfer-app,num_dependents_deps.dev:0
 github.com/zowe/imperative,
 github.com/zowe/jobs,num_dependents_deps.dev:0
-github.com/zowe/launcher,num_dependents_deps.dev:0
```
Comment (naveensrinivasan) [inline cron/internal/data/projects.csv:1243375]: What was the reason for these changes?

File: cron/internal/data/projects.csv [hunk 2]
```diff
@@ -1243392,8 +1243392,8 @@ github.com/zowe/zowe-cli-sample-plugin,num_dependents_deps.dev:0
 github.com/zowe/zowe-cli-scs-plugin,num_dependents_deps.dev:0
 github.com/zowe/zowe-cli-tasks-plugin,num_dependents_deps.dev:0
 github.com/zowe/zowe-cli-zos-make-plugin,
-github.com/zowe/zowe-client-python-sdk,
```
Comment (naveensrinivasan) [inline cron/internal/data/projects.csv:1243395]: The same applies here.

**Review by naveensrinivasan — COMMENTED** [review]
Thanks! LGTM.  Except for changes for projects.csv

Also, can you please include release notes in the release notes section?

**Review by naveensrinivasan — APPROVED** [review]

**Comment by naveensrinivasan** [comment]
@gabibguti  Please DCO the commit https://wiki.linuxfoundation.org/dco

**Comment by laurentsimon** [comment]
I updated the release-notes section

File: cron/internal/data/projects.csv (hunk 1 above)
Comment (gabibguti) [reply cron/internal/data/projects.csv:1243375]: When running `make all` the build process automatically made these changes in `projects.csv`. I will remove them.

File: cron/internal/data/projects.csv (hunk 2 above)
Comment (gabibguti) [reply cron/internal/data/projects.csv:1243395]: Same explanation.

**Comment by gabibguti** [comment]
> @gabibguti Please DCO the commit https://wiki.linuxfoundation.org/dco

All commits are signed-off now.

> I updated the release-notes section

Thanks laurent!

File: cron/internal/data/projects.csv (hunk 2 above)
Comment (naveensrinivasan) [reply cron/internal/data/projects.csv]: 👍 , Thanks

**Comment by laurentsimon** [comment]
DCO pre-submit has not started. Not sure why.

**Review by laurentsimon — APPROVED** [review]

**Review by justaugustus — APPROVED** [review]
Thanks @gabibguti!

**Review by naveensrinivasan — APPROVED** [review]

**Pull request merged**